conf/gear.conf

```diff
@@ -64,7 +64,7 @@ gearpump {
     enabled = true
 
     # We will take one metric out of ${sample.rate}
-    sample-rate = 10
+    sample-rate = 1
```

Comment:
1. With old sample-rate setting 10, when there a a few messages, < 10, the UI will display nothing. 
2. We heard complain before that the UI metrics is in-accurate on how many messages are received, as we will round it by 10.

3. Performance impact of this change still need investigation.